pkg/packagekit/updates.scss

```diff
@@ -351,3 +357,19 @@ table.header-buttons {
 table.available {
```

Comment:
This needs to be deleted.
 The margin right now is too much:
![Screen Shot 2020-12-07 at 10 12 04](https://user-images.githubusercontent.com/14921356/101331781-bccf5c00-3874-11eb-875c-498e85c6551d.png)